### PR TITLE
feat(eisp): sent hktm message directly to correct named destination

### PR DESCRIPTION
**Pull request by glmxndr**
Sorry @ilaborie ...

### CODE REVIEW

File: common/functions/src/main/java/com/csgroup/s2pdgs/functions/inventory/S2IPFInventoryFunction.java [hunk 1]
```diff
@@ -143,7 +143,7 @@ public FileObject pripDestination() {
                                         "Failed to store products in PRIP : %s".formatted(t.getMessage()));
                             }).flatMap(countPrip -> launchCatalogStorage(countPrip, desc))
                             .onSuccess(products -> logHelper.infoEnd(beginTs,
-                                    "Successfully ingested %d products in CATALOG".formatted(products)))
+                                    "Successfully ingested %d products in CATALOG".formatted(products.size())))
```
Comment (glmxndr) [inline common/functions/src/main/java/com/csgroup/s2pdgs/functions/inventory/S2IPFInventoryFunction.java:146]: (Unrelated but needed correction.)

Comment (ilaborie) [reply common/functions/src/main/java/com/csgroup/s2pdgs/functions/inventory/S2IPFInventoryFunction.java:146]: Do you want a specific PR?

Comment (glmxndr) [reply common/functions/src/main/java/com/csgroup/s2pdgs/functions/inventory/S2IPFInventoryFunction.java:146]: Just for this... not really.

**Comment by BenEsquis** [comment]
Ok for me,
Could you explain how to plug the named destination in the scdf stream definition ?
Thanks

**Comment by glmxndr** [comment]
> Ok for me, Could you explain how to plug the named destination in the scdf stream definition ? Thanks

`:${APP_NS}hktm-context > ...`

**Pull request merged**